qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/JavadocEmptyLineCheck/Invalid.java

```diff
@@ -0,0 +1,72 @@
+/**
+ *
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ */
+public final class Invalid {
+
+
+	private int x = 0;
+
+	/**
+	 *
+	 * empty javadoc line at the beginning
+	 */
+	private int y = 0;
+
+	/**
+	 * empty javadoc line at the end
+	 *
+	 */
+	private int z = 0;
+
+	/**
+	 *
+	 * empty javadoc line at the beginning
+	 */
+	public Invalid() {}
+
+	/**
+	 * empty javadoc line at the end
+	 *
+	 */
+	public void method() {
+	}
+
+	/**
+	 *
+	 * empty javadoc line at the beginning
+	 */
+	public void method2() {
+	}
+
+	/**
+	 *
+	 * empty javadoc line at the beginning
+	 */
+	public void method3() {
+	}
+
+	public void method4() {
+	}
+}
+/**
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ *
+ */
+interface I1 {
+	public void method3() {
```

Comment:
@dskalenko I don't see any reason to put things without Javadoc to `Invalid`. They are indeed invalid, but for different reason than we are testing: missing Javadoc